docs/sources/tempo/metrics-generator/span_metrics.md

```diff
@@ -125,9 +125,12 @@ func (ds RatioBasedSampler) Description() string {
 
 ### Filtering
 
-In some cases, you may want to reduce the number of metrics produced by the `spanmetrics` processor.
-You can configure the processor to use an `include` filter to match criteria that must be present in the span in order to be included.
-Following the include filter, you can use an `exclude` filter to reject portions of what was previously included by the filter policy.
+In some cases, you may want to reduce the number of metrics that are produced
+from the `spanmetrics` processor by using a filter. With a filter, you can
```

Comment:
Moving "use a filter" to the front of the sentence because it's the method people can use to reduce the number of metrics.
```suggestion
In some cases, you may want to use a filter to reduce the number of metrics that are produced
from the `spanmetrics` processor. With a filter, you can
```